main.go

```diff
@@ -15,137 +15,35 @@ import (
-		Platform:             os.Getenv("platform"),
-		Options:              os.Getenv("options"),
-
-		GemfilePath:          os.Getenv("gemfile_path"),
-		FastlaneVersion:      os.Getenv("fastlane_version"),
-		ITMSParameters:       os.Getenv("itms_upload_parameters"),
-	}
-}
-
-func (configs ConfigsModel) print() {
-	log.Infof("Configs:")
-
-	log.Printf("- IpaPath: %s", configs.IpaPath)
-	log.Printf("- PkgPath: %s", configs.PkgPath)
-
-	log.Printf("- ItunesconUser: %s", configs.ItunesconUser)
-	log.Printf("- Password: %s", input.SecureInput(configs.Password))
-	log.Printf("- AppPassword: %s", input.SecureInput(configs.AppPassword))
-
-	log.Printf("- AppID: %s", configs.AppID)
-	log.Printf("- BundleID: %s", configs.BundleID)
-	log.Printf("- SubmitForReview: %s", configs.SubmitForReview)
-	log.Printf("- SkipMetadata: %s", configs.SkipMetadata)
-	log.Printf("- SkipScreenshots: %s", configs.SkipScreenshots)
-	log.Printf("- SkipAppVersionUpdate: %s", configs.SkipAppVersionUpdate)
-	log.Printf("- TeamID: %s", configs.TeamID)
-	log.Printf("- TeamName: %s", configs.TeamName)
-	log.Printf("- Platform: %s", configs.Platform)
-	log.Printf("- Options: %s", configs.Options)
-
-	log.Printf("- GemfilePath: %s", configs.GemfilePath)
-	log.Printf("- FastlaneVersion: %s", configs.FastlaneVersion)
-	log.Printf("- ITMSParameters: %s", configs.ITMSParameters)
-}
-
-func (configs ConfigsModel) validate() error {
-	if configs.IpaPath == "" && configs.PkgPath == "" {
-		return errors.New("no IpaPath nor PkgPath parameter specified")
-	}
-
-	if configs.IpaPath != "" {
-		if err := input.ValidateIfPathExists(configs.IpaPath); err != nil {
-			return fmt.Errorf("IpaPath %s", err)
-		}
-	}
-
-	if configs.PkgPath != "" {
-		if err := input.ValidateIfPathExists(configs.PkgPath); err != nil {
-			return fmt.Errorf("PkgPath %s", err)
-		}
-	}
-
-	if err := input.ValidateIfNotEmpty(configs.ItunesconUser); err != nil {
-		return fmt.Errorf("ItunesconUser %s", err)
-	}
-
-	if err := input.ValidateIfNotEmpty(configs.Password); err != nil {
-		return fmt.Errorf("Password %s", err)
-	}
-
-	if configs.AppID == "" && configs.BundleID == "" {
-		return errors.New("no AppID or BundleID parameter specified")
-	}
-
-	if err := input.ValidateWithOptions(configs.SubmitForReview, "yes", "no"); err != nil {
-		return fmt.Errorf("SubmitForReview, %s", err)
-	}
-
-	if err := input.ValidateWithOptions(configs.SkipMetadata, "yes", "no"); err != nil {
-		return fmt.Errorf("SkipMetadata, %s", err)
-	}
-
-	if err := input.ValidateWithOptions(configs.SkipScreenshots, "yes", "no"); err != nil {
-		return fmt.Errorf("SkipScreenshots, %s", err)
-	}
-
-	if err := input.ValidateWithOptions(configs.SkipAppVersionUpdate, "yes", "no"); err != nil {
-		return fmt.Errorf("SkipAppVersionUpdate, %s", err)
-	}
-
-	if err := input.ValidateWithOptions(configs.Platform, "ios", "osx", "appletvos"); err != nil {
-		return fmt.Errorf("Platform, %s", err)
-	}
-
-	return nil
+// Configs ...
+type Configs struct {
```

Comment:
this should be unexported (the stepconf package prints it with title case anyway)

Reply:
Fixed in:
https://github.com/bitrise-io/steps-deploy-to-itunesconnect-deliver/pull/61/commits/e8197e70de7fab16f2e93b03a86a88197ba89ef0